.example.env

```diff
@@ -12,12 +12,12 @@ DATABASE_PORT=5432
 SERVER_MODE=production
 RECAPTCHA_PUBLIC_KEY=
 RECAPTCHA_PRIVATE_KEY=
-SMTP_EMAIL=
-SMTP_EMAIL_TLS=
 SMTP_HOST=
-SMTP_HOST_PASSWORD=
+SMTP_PORT=587
```

Comment:
This value won't need to be hardcoded since it will be assigned on staging

Reply:
This is just for local, settings for dev and staging are defined in the devops repo